### PR TITLE
Edit failover timing doc

### PR DESCRIPTION
**Pull request by Loquacity**
# Description

This builds on https://github.com/timescale/docs/pull/2470 which merged before I was done editing 😅  Just a line edit for wording and style.

# Links

Fixes #[insert issue link, if any]

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/about/latest/contribute-to-docs/)

# Review checklists

Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

*   [ ] Is the content technically accurate?
*   [ ] Is the content complete?
*   [ ] Is the content presented in a logical order?
*   [ ] Does the content use appropriate names for features and products?
*   [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

*   [ ] Is the content free from typos?
*   [ ] Does the content use plain English?
*   [ ] Does the content contain clear sections for concepts, tasks, and references?
*   [ ] Have any images been uploaded to the correct location, and are resolvable?
*   [ ] If the page index was updated, are redirects required
      and have they been implemented?
*   [ ] Have you checked the built version of this content?

### CODE REVIEW

**Review by ggodeke — APPROVED** [review]
Looks great! Thank you!

File: use-timescale/ha-replicas/high-availability.md [hunk 1]
```diff
@@ -114,112 +112,136 @@ alt="Diagram showing the primary failing, and the load balancer redirecting
...
-1.  Check which node is your primary again: `select * from pg_stat_replication;`
-    You should see a notice that your connection has been reset:
+
+1.  Schedule a switchover:
+
+    ```sql
+    CALL tscloud.cluster_switchover();
+    ```
+
+    By default, the switchover occurs in 30s. You can change the time by passing
```
Comment (Loquacity) [inline use-timescale/ha-replicas/high-availability.md:172]:
```suggestion
    By default, the switchover occurs in 30&nbsp;secs. You can change the time by passing
```

Had to pull out my Chicago guide for this one 😅

File: use-timescale/ha-replicas/high-availability.md [hunk 2]
```diff
@@ -81,15 +80,15 @@ alt="Creating a database replica in Timescale"
...
 
-Failover (and switchover) is the process of redirecting traffic from your
-primary to the HA replica within 15 seconds of the primary becoming unresponsive.
-As part of failover, the HA replica is “promoted” to become the new primary.
-This process is invisible to the end user except for a connection reset. In the
-background, a new replica is immediately provisioned for the new primary.
+Failover is the process of redirecting traffic from your primary to the HA
+replica within 15 seconds of the primary becoming unresponsive. As part of
+failover, the HA replica is promoted to become the new primary. You should not
+notice this process, except for a connection reset. In the background, a new
```
Comment (Rajakavitha1) [inline use-timescale/ha-replicas/high-availability.md:86]:
```suggestion
failover, the HA replica is promoted to become the new primary and you will notice a connection reset. In the background, a new
```

**Review by Rajakavitha1 — APPROVED** [review]
LGTM!!!!

**Pull request merged**